src/fmod_cache.cpp

```diff
@@ -128,19 +128,43 @@ bool FmodCache::has_event_path(const String& eventPath) {
 }
 
 Ref<FmodVCA> FmodCache::get_vca(const FMOD_GUID& guid) {
-    return vcas.get(guid);
+    if (
+            HashMap<FMOD_GUID, Ref<FmodVCA>, FmodGuidHashMapHasher, FmodGuidHashMapComparer>::Iterator iterator{
+                    vcas.find(guid)
+            }
+    ) {
+        return iterator->value;
+    }
+
+#ifdef DEBUG_ENABLED
+    GODOT_LOG_WARNING(vformat("Cannot find vca with guid: %s", fmod_guid_to_string(guid)));
+#endif
+
+    return {};
 }
 
 Ref<FmodVCA> FmodCache::get_vca(const String& vcaPath) {
-    return vcas.get(strings_to_guid.get(vcaPath));
+    return get_vca(strings_to_guid.get(vcaPath));
```

Comment:
Don't you need to all check if the string has a matching guid too as well ? Otherwise you will end up with the same kind of crash.

Reply:
true